### ISSUE TITLE
Support for pasting images when filing an issue

### ISSUE THREAD

**Issue by evancharlton** [codetree/feedback#295]
When filing a GitHub issue, you can paste an image directly into the field. I don't know what magic you'd need to do to make this work seamlessly, but ..  please do it :-)

**Comment by ac21** [comment]
@evancharlton - can you expand on your comment a bit?  Does it work sporadically for you?  Also, what browser and version are you using?  I just tested this out and seems to work in the latest chrome but appears a bit flakey in Firefox;  if you can corroborate, I'll see what we can do to get this working a bit better for our FF friends...

**Comment by evancharlton** [comment]
Interesting. I was trying in Chrome and it didn't work -- I'll try again though.

**Comment by evancharlton** [comment]
I'm using modern Chrome and I'm completely unable to get this to work. Concrete repro steps:
1. Take a screenshot using `<cmd>+<control>+<shift>+<4>` (this allows you to select a region and puts the PNG on your paste buffer)
2. Open Codetree -> [new issue]
3. Put focus on the body area
4. Push `<cmd>+<v>`

Nothing happens. However, if I push `<cmd>+<v>` right now, I get ...
![image](https://user-images.githubusercontent.com/418560/52234484-0e49e100-28c2-11e9-873b-8ec7d1ab8edf.png)

^ this gets automatically uploaded & embedded

**Comment by ac21** [comment]
@evancharlton - The `<cmd>+<control>+<shift>+<4>` functionality also saves a file locally somewhere, correct?  Can you drag and drop that same image into the issue comment or main body?

When you drop the image into the comment or description box, you should get something like this:

![Screen Shot 2019-02-05 at 11.17.36 PM.png](https://codetreeusercontent.s3.amazonaws.com/uploads/d34fd401-b5e8-4f98-b486-35be26b7f023/Screen+Shot+2019-02-05+at+11.17.36+PM.png)

**Comment by evancharlton** [comment]
No, it just puts it on the buffer (which is incredibly useful). You're thinking of `<cmd>+<shift>+<4>`, which saves it to the desktop (leading to a bunch of trash that I have to clean up later)
